pkg/flavor/kubernetes/nfs.go

```diff
@@ -26,7 +26,7 @@ const (
 	defaultNFSNamespace = "hpe-nfs"
 	defaultNFSImage     = "hpestorage/nfs-provisioner:2.8.3-4"
 
-	creationInterval           = 60 // 120s with sleep interval of 2s
+	creationInterval           = 150 // 300s with sleep interval of 2s
```

Comment:
instead of 150 tries, would it be better to sleep for 5 seconds instead of 2 ?

Reply:
sounds good as deployment usually takes more than 5s initially to start. changed it to 5s interval.